duckduckgo/Android
# Feature/ui tidy up

Task/Issue URL: https://app.asana.com/0/inbox/471012189846018/593495966844380/592667188539744
Tech Design URL: 
CC: 

**Description**:
There were a few big UI changes that happened since the last release, including the consolidation of the landing page and the introduction of tabs.

This task is about tidying up some of the rough edges around the UI that were either introduced during these features, or that couldn't be easily fixed until they were completed.


**Steps to test this PR**:
1. Test the animation on the DDG logo on the landing page.
    - Fresh app launch
    - Returning to running app
    - Launching using widget
    - Launching with assist intent
    - Returning to landing page from webview
    - Returning to landing page from settings/bookmarks/tabs screen
1. The toolbar shouldn't shrink in width (which was happening sometimes; though not easily reproducible
1. Keyboard should generally appear when you want it to and not when you don't.
    - If it ever surprises you, make a note of it and we'll compare expectations or determine if it's a bug.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
